openbas-model/src/main/java/io/openbas/database/repository/ExerciseRepository.java

```diff
@@ -88,4 +111,37 @@ public interface ExerciseRepository extends CrudRepository<Exercise, String>,
             "WHERE exercises.exercise_created_at < :from " +
             "AND users_groups.user_id = :userId ;", nativeQuery = true)
     Iterable<RawGlobalInjectExpectation> rawGrantedInjectExpectationResultsFromDate(@Param("from") Instant from, @Param("userId") String userId);
+
+    /**
+     *  Get the raw version of the exercises
+     * @return the list of exercises
+     */
+    @Query(value = " SELECT ex.exercise_category, ex.exercise_id, ex.exercise_status, ex.exercise_start_date, ex.exercise_name, " +
+            " ex.exercise_subtitle, array_agg(et.tag_id) FILTER ( WHERE et.tag_id IS NOT NULL ) as exercise_tags, " +
+            " array_agg(injects.inject_id) FILTER ( WHERE injects.inject_id IS NOT NULL ) as inject_ids " +
+            "FROM exercises ex " +
+            "LEFT JOIN injects_expectations ie ON ex.exercise_id = ie.exercise_id " +
+            "LEFT JOIN injects ON ie.inject_id = injects.inject_id " +
```

Comment:
I didn't know you could omit the variable name from the table
LEFT JOIN injects X ON ...